.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+# This configuration file was mainly inspired by ./doc/SETUP_DEVELOPMENT_DOCKER.md
+
+tasks:
+  - name: Rails
+    init: |
+      # install correct ruby+tools versions
+      rvm install 2.6.9 # should be similar as in /Dockerfile-dev 
+      rvm use 2.6.9
+      gem install bundler:1.17.3
```

Comment:
Would `gem install bundler` suffice - or is there a dependency on the specific version? I'd hope to make Ruby version upgrades somewhat smooth in this way.

Reply:
Yep, these are all great suggestions. I've integrated them in the current version of the PR.